charts/kubernetes-agent/values.yaml

```diff
@@ -13,6 +13,7 @@ agent:
   space: "Default"
   targetEnvironments: []
   targetRoles: []
+  machinePolicy: ""
```

Comment:
would `machinePolicyName` be more accurate/clearer?

Reply:
I wasn't sure because Environments and Space also match by name but we don't specify it 🤔

Reply:
I think let's be specific if we can

Reply:
Done 👍